participant-datastore/consent-mgmt-module/consent-mgmt/src/main/java/com/google/cloud/healthcare/fdamystudies/MyStudiesUserRegWsApplication.java

```diff
@@ -13,7 +13,9 @@
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 import org.springframework.boot.autoconfigure.domain.EntityScan;
 import org.springframework.boot.autoconfigure.orm.jpa.HibernateJpaAutoConfiguration;
+import org.springframework.scheduling.annotation.EnableScheduling;
```

Comment:
Please update file copyright to `2020-2021`

Reply:
Fixed review comment.